app/components/Sidebar/components/Version.tsx

```diff
@@ -37,10 +39,10 @@ export default function Version() {
           <br />
           <LilBadge>
             {releasesBehind === 0
-              ? "Up to date"
-              : `${releasesBehind} version${
-                  releasesBehind === 1 ? "" : "s"
-                } behind`}
+              ? t("Up to date")
+              : `${releasesBehind} ` +
+                (releasesBehind === 1 ? t("version") : t("versions")) +
+                t("behind")}
```

Comment:
```suggestion
              : t(`{{ releasesBehind }} versions behind`, {
                  releasesBehind,
                  total: releasesBehind, 
                })}
```

More like this, will generate the plural and non-plural strings in translation files

Reply:
OK. But I didn't understand how you'll manage the non-plural strings with your suggestion ?

Reply:
The translation key will be generated with and without a _plural suffix automatically.

Reply:
This is thanks to the `total: releasesBehind` line ?